CMakeLists.txt

```diff
@@ -143,6 +144,7 @@ list(APPEND _fv3atm_defs_private GFS_PHYS
 target_compile_definitions(fv3atm PRIVATE "${_fv3atm_defs_private}")
 
 set_target_properties(fv3atm PROPERTIES Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/mod)
+target_include_directories(fv3atm PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
Not needed anymore.

Reply:
Commit 7d331b4f3883dac990eaae1d4f7a45768027ed1d removes the no longer needed line. Thanks @DusanJovic-NOAA!